dev/io.openliberty.grpc.1.0.internal/src/io/grpc/servlet/ServletServerStream.java

```diff
@@ -248,6 +250,18 @@ public void writeHeaders(Metadata headers) {
     public void writeFrame(
     		@Nullable 
     		WritableBuffer frame, boolean flush, int numMessages) {
+      
+      MessageFramer framer = ServletServerStream.this.framer();
+      if( framer != null && framer.isClosed() && !flush ) {
```

Comment:
link

Reply:
Link?

Reply:
It is what I write in order to get a URL into a PR change. I think I used it in a  slack message.